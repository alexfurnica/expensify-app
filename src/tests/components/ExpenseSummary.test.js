import React from 'react'
import {ExpenseSummary} from '../../components/ExpenseSummary'
import {shallow} from 'enzyme'

test('Should render with 1 expense', () => {
  const wrapper = shallow(<ExpenseSummary expenseCount={1} expensesTotal={235} />)

  expect(wrapper).toMatchSnapshot()
})

test('Should render with 2 expenses', () => {
  const wrapper = shallow(<ExpenseSummary expenseCount={20} expensesTotal={23487200} />)

  expect(wrapper).toMatchSnapshot()
})