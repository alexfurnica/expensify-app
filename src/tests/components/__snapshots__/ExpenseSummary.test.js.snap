// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`Should render with 2 expenses 1`] = `
<div>
  <h1>
    Viewing 
    20
     
    expenses
     totalling 
    $234,872.00
  </h1>
</div>
`;
